pkgs/tools/misc/pre-commit-hook-ensure-sops/default.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, fetchpatch
+}:
+
+python3Packages.buildPythonPackage rec {
```

```suggestion
python3Packages.buildPythonApplication rec {
```

It does not need to propagate its dependencies, since it is an application.